cns/logger/heartbeat.go

```diff
@@ -1,28 +1,91 @@
-// Copyright Microsoft. All rights reserved.
+// Copyright 2018 Microsoft. All rights reserved.
```

Comment:
I think we are not adding the year. That would need us to update those every year ( when we touch those files )